Changelog.md

```diff
@@ -4,6 +4,14 @@
 
 -   Enable installing older app versions.
 
+### Fixed
+
+-   PPK-1 measurements fix.
```

Comment:
I believe we don't tend to mention app specific fixes in the launcher changelog (despite the fix being applied here).

If it really has to be mentioned, then please be more specific about what is being fixed.

Reply:
Got approval from Bjorn Inge